README.md

```diff
@@ -13,6 +13,16 @@ npm install
 npm run dev
 ```
 
+## Run functions locally
+
+Run a local build to run the functions using the following command:
+
+```
+netlify dev
+```
+
+> Make sure you have the correct path to `POSTGRES_URL` in your local `.env` file to test the functions
+
```

Comment:
More changes that should be merged into the epic/prisma-3 branch so they are not listed here. (There are many more now)

Reply:
I literally merged the changes form the epic into this branch a minute ago

Reply:
The changes that have followed the last commit are form me merging the latest from the epic into this branch. But thats not really how it should work as they changes should match... Hmm

Reply:
I think you fixed it!

Reply:
![phew](https://user-images.githubusercontent.com/22930449/130210665-a8ae921e-f60b-4a84-9d2a-c7705b80ddb4.gif)